Kitodo/src/main/resources/kitodo_projects.xml

```diff
@@ -131,16 +130,13 @@
         </createNewProcess>
         <tifheader>
             <monograph>
-                '|[[DOC_TYPE]]'+$Doctype+'|[[HAUPTTITEL]]'+Titel+'|[[AUTOREN/HERAUSGEBER]]'+Autoren+'|[[JAHR]]'+Erscheinungsjahr+'|[[ERSCHEINUNGSORT]]'+Erscheinungsort+'|[[VERZ_STRCT]]'+ATS+'_'+PPN
-                digital a-Satz+'|'
+                '|[[DOC_TYPE]]'+$Doctype+'|[[HAUPTTITEL]]'+TitleDocMain+'|[[AUTOREN/HERAUSGEBER]]'+Autoren+'|[[JAHR]]'+PublicationYear+'|[[ERSCHEINUNGSORT]]'+PlaceOfPublication+'|[[VERZ_STRCT]]'+TSL_ATS+'_'+CatalogIDDigital+'|'
```

Comment:
Why didn't you change "Erscheinungsort" to english? Or the others?

Reply:
Instead of using the labels in the title definition and tifHeader definition I have used the metadataIds.